styles.css

```diff
@@ -2,6 +2,149 @@
 
```

Comment:
add scroll-behaviour: smooth to html element for smooth scrooling.

Reply:
I have made the changes. Thanks for reviewing my code.